designs/2022-community-eslint-org/README.md

```diff
@@ -0,0 +1,328 @@
+    change for them? If so, how are you going to minimize the disruption
+    to existing users?
+-->
+
+It won't affect existing ESLint users, as package could be released under the
+same name on `npm`.  
+Even when we would release a package under the `@eslint-community` npm scope
+(for instance because we can't get the rights to publish to the existing npm
+package), people could still use the old package without any extra problems
+(except for the ones they already had) if they wanted to.
+
+## Alternatives
+
+<!--
+    What other designs did you consider? Why did you decide against those?
+
+    This section should also include prior art, such as whether similar
+    projects have already implemented a similar feature.
+-->
+
+- Private persons could create a fork of the above mentioned packages.  
+  This could however create ceveral well maintained forks, which would cause
+  fragmentation in the community.
+
+  More importantly, this could cause the "problem" of the fork being reliant on
+  that specific individual, which could put us in the same position
+  (abandoned/unmaintained package) all over again at some point in time.
+  Possibly having a new fork each X amount of time, would also have the burden
+  of needing to update all existing dependants each time.
+
+- The community could independently create multiple more focused organizations
+  to maintain a subset of ESLint related packages.  
+  Some examples of already existing organizations like that are
+  [@import-js](https://github.com/import-js) (which maintains
+  [`eslint-plugin-import`](https://github.com/import-js/eslint-plugin-import)),
+  [@jsx-eslint](https://github.com/jsx-eslint) (which maintains
+  [`eslint-plugin-jsx-a11y`](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y),
+  [`eslint-plugin-react`](https://github.com/jsx-eslint/eslint-plugin-react) &
+  [`jsx-ast-utils`](https://github.com/jsx-eslint/jsx-ast-utils)) &
+  [@standard](https://github.com/standard) (which maintains a couple of
+  [`standard`](https://github.com/standard/standard) related ESLint configs)
+
+## Open Questions
+
+<!--
+    This section is optional, but is suggested for a first draft.
+
+    What parts of this proposal are you unclear about? What do you
+    need to know before you can finalize this RFC?
+
+    List the questions that you'd like reviewers to focus on. When
+    you've received the answers and updated the design to reflect them,
+    you can remove this section.
+-->
+
+- What exact acceptance criteria do we want to use?
+- What's the main place of communication for the "core" team?
+- Will there be a single team collectively maintaining all modules within the
+  project? Or will each module possibly have their own maintainers?
+- What maintenance principles will be enforced?  
+  Will each module have their own standards, release flows, dependency update
+  checks etc or will it be streamlined across all modules?
```

Comment:
it seems awfully hard to have the same tools to meet all the projects - they were created for different goals, maintained by different people, having different code structure....

I'd suggest a loose mode - the maintainers are free to use the tools - based how it was working before.

Reply:
It shouldn't be obligated right away, but I think it should be the goal to get some consistency in all the packages.

Especially when talking about supported Node/ESLint versions, publish workflow, ... I think this would be a good idea to make it consistent.

Reply:
If it’s not up to the maintainers, then they’ll be discouraged from contributing their project. There’s a reason that foundations like OpenJS mandate almost nothing for member projects.

Reply:
yes, it's not a common org like standard; more like openJS - a collection of projects.